drivers/clock_control/clock_stm32_ll_u5.c

```diff
@@ -561,9 +564,6 @@ int stm32_clock_control_init(const struct device *dev)
 		return r;
 	}
 
-	/* Set voltage regulator to comply with targeted system frequency */
-	set_regu_voltage(CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC);
-
 	/* Set flash latency */
 	/* If freq increases, set flash latency before any clock setting */
```

Comment:
I believe the configuration of the flash latency, in case old < new, should be moved up as well.

Reply:
OK, to set flash latentcy before _any_ clock setting
(the refMan says to "Adjust number of wait states according new frequency" before the PLL config.)